docs/auth.md

```diff
@@ -248,6 +248,60 @@ generated containing the credentials configured in the `Secret`, and these
 credentials are then used to authenticate when retrieving any
 `PipelineResources`.
 
+## Kubernetes's Docker registry's secret
+
+Kubernetes defines two types of secrets for Docker registries :
+`kubernetes.io/dockercfg` and `kubernetes.io/dockerconfigjson`. Tekton supports
+those secrets in addition to the one described above.
+
+1. Define a `Secret` from a Docker client configuration file, as documented in
+   [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
+
+   ```bash
+   kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+   ```
+
+1. Direct a `ServiceAccount` to use this `Secret`:
+
+   ```yaml
+   apiVersion: v1
+   kind: ServiceAccount
+   metadata:
+     name: build-bot
+   secrets:
+     - name: regcred
+   ```
+
+1. Use that `ServiceAccount` in your `Build`:
```

Comment:
Should this be a `Task`?

Reply:
:man_facepalming: @dibyom you are right ! :sweat_smile: :joy: